src/main/java/com/jvm_bloggers/entities/blog/Blog.java

```diff
@@ -73,7 +83,7 @@ public boolean isCompany() {
     }
 
     public Boolean getDefaultApprovedValue() {
-        return isPersonal() ? Boolean.TRUE : null;
+        return !isModerationRequired();
```

Comment:
Maybe we could use `isModerationRequired()` method directly and remove this one?